modules/go/01_mod.mk

```diff
@@ -57,27 +57,24 @@ generate-go-mod-tidy: | $(NEEDS_GO)
 
 shared_generate_targets += generate-go-mod-tidy
 
-ifndef govulncheck_skip
+base_dir := $(dir $(lastword $(MAKEFILE_LIST)))/base/
 
-default_govulncheck_generate_base_dir := $(dir $(lastword $(MAKEFILE_LIST)))/base/
-# The base directory used to copy the govulncheck GH action from. This can be
-# overwritten with an action with extra authentication or with a totally different
-# pipeline (eg. a GitLab pipeline).
-govulncheck_generate_base_dir ?= $(default_govulncheck_generate_base_dir)
```

Comment:
This was not used, use `dont_generate_govulncheck` instead and manually maintain the yaml file.